src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsTag.cs

```diff
@@ -57,13 +57,20 @@ public override GraphicsResult GetGraphics(IWpfTextView view, Geometry bounds, T
             };
 
             var idRun = GetRunForId(out var hyperlink);
-
+            var editorBackground = (Color)_editorFormatMap.GetProperties("TextView Background")["BackgroundColor"];
             if (hyperlink is null)
             {
                 block.Inlines.Add(idRun);
             }
             else
             {
+                // If we are in dark mode, then we want to change the color of the link to be lighter than the
+                // default color to meet accessibility standards.
+                if (editorBackground == Color.FromArgb(255, 30, 30, 30))
+                {
+                    hyperlink.Foreground = new SolidColorBrush(Color.FromRgb(3, 194, 252));
```

Comment:
should I create a new editor format definition just for the hyperlink colors?

Reply:
There should already be one, which is used e.g. for hyperlinks in Quick Info. Does it not work here?

Reply:
https://github.com/dotnet/roslyn/blob/d3e915d2dc49f4371f510add785ea011ca6c36be/src/EditorFeatures/Core/Implementation/IntelliSense/Helpers.cs#L134 

For reference. This looks to be how quickinfo does it

Reply:
@ryzngard I debugged on that line but it does not hit the links there, that is only for determining the classification of the message I believe.

Reply:
https://github.com/dotnet/roslyn/blob/d3e915d2dc49f4371f510add785ea011ca6c36be/src/EditorFeatures/Core/Implementation/IntelliSense/Helpers.cs#L142-L144

Those are the important lines. It uses https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.text.adornments.classifiedtextrun?view=visualstudiosdk-2019 , although I'm not sure if that's usable here. @sharwell might know more

Reply:
yeah I tried that, it was not usable with how I am doing things. I did change it to just use what is set in the classification format map.